coops-ui/tests/unit/DirectorListSm.spec.ts

```diff
@@ -59,12 +62,90 @@ describe('DirectorListSm', () => {
       }
     ]
 
-    const wrapper = mount(DirectorListSm, { store })
+    const wrapper = mount(DirectorListSm, { store, vuetify })
+    const vm = wrapper.vm as any
+
+    Vue.nextTick(() => {
+      expect(vm.directors.length).toEqual(2)
+      expect(vm.directors[0].mailingAddress).toBeUndefined()
+      expect(vm.$el.querySelectorAll('.address-panel').length).toEqual(2)
+
+      wrapper.destroy()
+      done()
+    })
+  })
+
+  it('handles empty data as a coop', done => {
+    // init store
+    store.state.directors = []
+    store.state.entityType = EntityTypes.Coop
+
+    const wrapper = mount(DirectorListSm, { store, vuetify })
+    const vm = wrapper.vm as any
+
+    Vue.nextTick(() => {
+      expect(vm.directors.length).toEqual(0)
+      expect(vm.directors.mailingAddress).toBeUndefined()
+      expect(vm.$el.querySelectorAll('.address-panel').length).toEqual(0)
+
+      wrapper.destroy()
+      done()
+    })
+  })
+
+  it('displays multiple directors as a bcorp', done => {
+    // init store
+    store.state.entityType = EntityTypes.BCorp
+    store.state.directors = [
+      {
+        'officer': {
+          'firstName': 'Peter',
+          'lastName': 'Griffin'
+        },
+        'deliveryAddress': {
```

Comment:
Can you please add a unit test to confirm that "Same as above" is displayed when the mailing address is the same as the delivery address?

Reply:
Absolutely, will do!

Reply:
Done!